projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+---
+stoplight-id: 73ef6e15ee3ec
+---
+
+# Project flow from test to production
+
+For the integration with the UiTdatabank Entry API, there is a flow in which you first obtain access to our test environment. 
+Access to production environment is given after your integration on test environment has been successful validated by publiq vzw.
+
+On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!
+
+> Make sure you read this guide before you start developing on our test environment!
+
+## Step 1: start on test environment
+
+You can apply for free test credentials at [Projectaanvraag](https://projectaanvraag.uitdatabank.be). 
```

Comment:
General note to ourselves: When publiq platform goes live we should do a search on links to projectaanvraag.uitdatabank.be. We can of course add a redirect, but it would be better to also update the link text (as "Projectaanvraag" won't exist anymore).